### ISSUE TITLE
Update or remove gulp-util dependency

### ISSUE THREAD

**Issue by dwighthouse** [intervalia/gulp-hash-filename#6]
`gulp-util` has been deprecated. When attempting to install gulp-hash-filename, the following message is displayed:

```
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
```

From what I can see in the codebase, you aren't actually using it. There are four locations where you import it, but then you never use the imported `gutil` variable.

**Comment by dwighthouse** [comment]
PR created: https://github.com/intervalia/gulp-hash-filename/pull/7

**Comment by intervalia** [comment]
Resolved in Version 2.0.1
